templates/problem/editorial.html

```diff
@@ -19,8 +19,11 @@
 {% block body %}
     <div class="content-description">
         {% if not has_solved_problem %}
-            <div class="alert alert-danger">{{ _('Remember to use this editorial <b>only</b> when stuck, and <b>not to copy-paste code from it</b>. Please be respectful to the problem author and editorialist. <br><br> <b>Submitting an official solution before solving the problem yourself is a bannable offence.</b>') }}</div>{% endif %}
-        <p>Authors: {{ link_users(solution.authors.all()) }}</p>
+            <div class="alert alert-danger">{{ _('Remember to use this editorial <b>only</b> when stuck, and <b>not to copy-paste code from it</b>. Please be respectful to the problem author and editorialist. <br><br> <b>Submitting an official solution before solving the problem yourself is a bannable offence.</b>') }}</div>
+        {% endif %}
+        {% if solution.authors.exists() %}
+            <p>Authors: {{ link_users(solution.authors.all()) }}</p>
```

Comment:
Solution inefficient. This runs two queries. You should just do a with on the `all` queryset, and use `if` on that.